virt/virt-2-4-release-notes.adoc

```diff
@@ -147,8 +147,26 @@ $ oc delete pod -n openshift-cnv --selector=kubevirt.io=virt-handler --field-sel
 ----
 <1> Where <source-node-name> is the name of the source node that the VMI is migrating from.
 
-//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
-* If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
+* If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859235[*BZ#1859235*])
+
+[NOTE]
+====
+Virtual machines that were created from a common template prior to upgrade might also need to be patched with an updated `terminationGracePeriodSeconds`.
```

Comment:
Because you don't mention `terminationGracePeriodSeconds` until after this note, I'd change this to something even simpler, like: 
```
[NOTE]
====
You must also patch any virtual machines that were created from common templates prior to upgrade.
====
```
(I'd suggest adding `value` after `terminationGracePeriodSeconds` if you don't take that suggested rewrite.)